code/game/machinery/defibcharger.dm

```diff
@@ -87,8 +87,6 @@ obj/machinery/recharger/defibcharger/wallcharger/process()
 	return -1
 
 obj/machinery/recharger/defibcharger/wallcharger/attackby(obj/item/weapon/G as obj, mob/user as mob)
-	if(..())
```

Comment:
*buzzer

Wrong, the if(..()) return also checks for powered/unpowered and broken